packages/fluere/src/core/executor.ts

```diff
@@ -397,6 +397,22 @@ export function createExecutor<Start, Stop>(
     while (true) {
       const currentEventData = queue.shift();
 
+      // FIXME: REMOVE THIS IN THE FUTURE!
+      //  THIS IS A TEMPORARY FIX FOR THE EMPTY QUEUE
+      function fallbackFoundMissing(executor: InternalExecutorContext) {
+        const events = executor.__internal__currentEvents;
+        events.forEach((ev) => {
+          if (!enqueuedEvents.has(ev)) {
+            // todo: should warn user?
+            queue.push(ev);
+          }
+        });
+        for (const next of executor.next) {
+          fallbackFoundMissing(next);
+        }
+      }
+      fallbackFoundMissing(rootExecutorContext);
```

Comment:
Idk should we use process.nextTick in the future?

For now I think it's not possible to detect whether a function is used in handler.